core/src/env_platform/env_service.bal

```diff
@@ -107,6 +198,33 @@ service envservice on ep0 {
     }
     resource function assignMinistry(http:Caller caller, http:Request req, string applicationId, AssignedMinistry body) returns error? {
 
+        http:Response response = new;
+        string authHeader = req.getHeader("Authorization");
+        [string, string]|error adminInfoFromJWT = getUserInfoFromJWT(authHeader);
+        if (adminInfoFromJWT is [string, string]) {
+            [string, string] [adminId, adminType] = adminInfoFromJWT;
+            log:printDebug("User information - admin ID: " + adminId + ", admin type: " + adminType + ".");
+            boolean|error assignMinistryResult = assignMinistry(body, applicationId, adminId);
+            if (assignMinistryResult is boolean && assignMinistryResult) {
+                log:printDebug("Successfully assigned the ministry " + body.ministry.name + " for the application with ID: " + applicationId + ".");
```

Comment:
Check the length of the line. Breakdown if it's greater than 120. Fix this on the other places as well.

Reply:
Resolved this and other places as well.